detox/ios/Detox/DetoxInstrumentsManager.m

```diff
@@ -55,6 +55,22 @@ typedef NS_ENUM(NSUInteger, __DTXEventStatus) {
 WEAK_IMPORT_ATTRIBUTE
 @interface DTXProfiler : NSObject @end
 
+@interface DetoxInstrumentsConfiguration ()
```

Comment:
Is `DetoxInstrumentsConfiguration` needed if all it does is hold the DTXProfilerConfiguration object? Why not just return that and pass it to recording manager?

Reply:
Because we'll need to specify the configuration type somehow. If we'll just pass the `id` type so anyone outside may pass the any id into it.

Reply:
So why not make the entry method startReacordingWithDictionary: and internally create the configuration?

Reply:
Better to separate configuration creation from starting/continue logic.